### PR TITLE
Initial work to get a simple CLI going for creating clubhouse tickets

### PR DESCRIPTION
**Pull request by assiotis**
A quick job to get a simple CLI going so it is easier for backend devs to create tickets in Clubhouse without having to click around in the UI

### CODE REVIEW

File: src/clubhouse.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  state: string;
+  position: number;
+  started: boolean;
+  started_at?: Date;
+  started_at_override?: any;
+  completed: boolean;
+  completed_at?: Date;
+  completed_at_override?: any;
+  archived: boolean;
+  labels: any[];
```
Comment (springuper) [inline src/clubhouse.ts:38]: could you use a specific type instead of `any`

File: src/clubhouse.ts [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+}
+
+export function getUsers(token: string): Promise<IMember[]> {
+  return got(`${API_BASE_URL}/members`, {json: true, query: {token}})
+    .then(response => response.body as IMember[]);
+}
+
+export function createStory(token: string, story: IStory): Promise<IStory> {
+  const client = require('clubhouse-lib').create(token);
+  const reqObj: IStory = {
```
Comment (springuper) [inline src/clubhouse.ts]: Ramda’s `pick` can save some duplications

File: src/clubhouse.ts [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+export function getState(cfg: IConfiguration): Promise<IClubhouseState> {
+  const token = cfg.token;
+  return Promise
+    .all([getProjects(token), getEpics(token), getUsers(token)])
+    .then(data => ({
+      projects: data[0],
+      epics: data[1],
+      users: data[2],
+      configuration: cfg,
+      loaded: true,
```
Comment (springuper) [inline src/clubhouse.ts:175]: why use this flag? if this promise is resolved, we can know it from a following `then`

File: src/index.ts [hunk 4]
```diff
@@ -0,0 +1,136 @@
+import {Answers,  Question, Questions} from "inquirer";
```
Comment (springuper) [inline src/index.ts]: reminds be extra white spaces

File: src/index.ts [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+      inquirer
+        .prompt({
+          name: 'doOneMore',
+          type: 'confirm',
+          message: `Create another ticket: `,
+          default: true,
+        })
+        .then((res: Answers) => {
+          if (res.doOneMore) {
+            idx += 1;
```
Comment (springuper) [inline src/index.ts]: why plus 1 twice?

File: src/index.ts [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+            .filter((user: ChoiceOption) => user.name.toLowerCase().includes(input.toLowerCase()));
+          return Promise.resolve(filtered);
+        }
+      },
+    }] as Question[];
+
+  return inquirer
+    .prompt(questions)
+    .then((res: any) => {
+      createStory(state.configuration.token, {
```
Comment (springuper) [inline src/index.ts]: return?

File: src/clubhouse.ts (hunk 1 above)
Comment (assiotis) [reply src/clubhouse.ts:38]: don't really need this for now

**Pull request merged**